### PR TITLE
Only duplicate editions when creating a new draft

### PR DESCRIPTION
**Pull request by AndrewVos**
- Comments, and approvals, will stick around on the edition even when it is published.
- Only published versions of editions will stay around forever
- The code in GuidesController should be a lot easier to understand
- I have used inline styles for the "Create a new edition" form for now. Sorry.

### CODE REVIEW

**Comment by tadast** [comment]
It would be nice to explain why we are going back to this behaviour and what are the implications of this change in the PR description.

**Comment by AndrewVos** [comment]
@tadast ready for rereview

File: app/views/guides/index.html.erb [hunk 1]
```diff
@@ -23,7 +23,13 @@
           <%= state_label(guide) %>
         </td>
         <td class='content-controls'>
-          <%= link_to edit_action_label(guide), edit_guide_path(guide), class: 'btn btn-default btn-xs' %>
+          <% if guide.latest_edition.published? %>
+            <%= form_for :drafts, url: guide_drafts_path(guide_id: guide.id), html: {style: "display: inline;"} do |f| %>
```
Comment (tadast) [inline app/views/guides/index.html.erb]: :laughing: I guess inline styles will go away when we have the actual designs :)

Comment (AndrewVos) [inline app/views/guides/index.html.erb]: Yes they will!

File: spec/features/guide_publishing_flow_spec.rb [hunk 2]
```diff
@@ -7,59 +7,79 @@
...
-  it "should create a new edition even when saving a draft" do
-    guide = given_a_guide_exists state: 'draft'
-    visit guides_path
-    link = there_should_be_a_control_link "Continue editing", document: guide
-    link.click
-    fill_in "Title", with: "Sample Published Edition 2"
-    click_button "Save Draft"
-    expect(current_path).to eq root_path
+  context "latest edition is not published" do
+    it "should not create a new draft edition" do
```
Comment (tadast) [inline spec/features/guide_publishing_flow_spec.rb]: "should not save an extra draft if someone else clicks the link in the meantime" ?

Comment (AndrewVos) [inline spec/features/guide_publishing_flow_spec.rb]: Done

**Pull request merged**